app/views/providers/visas/edit.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= content_for :page_title, "#{@form_object.errors.present? ? 'Error: ' : ''}Visa sponsorship" %>
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(details_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year)) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(
+      model: @form_object,
+      url: provider_recruitment_cycle_visas_path(@provider.provider_code, @provider.recruitment_cycle_year),
+      method: :post,
+      scope: "",
+      builder: GOVUKDesignSystemFormBuilder::FormBuilder,
+    ) do |form| %>
+
+      <%= form.govuk_error_summary %>
+
+      <h1 class="govuk-heading-xl">
+        <span class="govuk-caption-xl"><%= @provider.provider_name %></span>
```

Comment:
Have now updated heading sizes to be one size smaller, so need to update this to match.

```suggestion
      <h1 class="govuk-heading-l">
        <span class="govuk-caption-l"><%= @provider.provider_name %></span>
```